awscrt/io.py

```diff
@@ -83,6 +94,31 @@ def on_shutdown():
         self.shutdown_event = shutdown_event
         self._binding = _awscrt.event_loop_group_new(num_threads, is_pinned, cpu_group, on_shutdown)
 
+    @staticmethod
+    def get_or_create_static_default():
+        global _static_event_loop_group
+        if '_static_event_loop_group' in globals():
+            if isinstance(_static_event_loop_group, EventLoopGroup):
+                return _static_event_loop_group
```

Comment:
gotta use locks when checking the singleton 

otherwise it's theoretically possible for two threads to call this at once, both threads see that no singleton exists, and both threads go ahead and create one